Management/src/test/java/com/stack/jspdemo/ManagementApplicationTests.java

```diff
@@ -0,0 +1,16 @@
+package com.stack.jspdemo;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class ManagementApplicationTests {
```

Comment:
If your tests aren't ready to be updated avoid pushing empty tests.

Reply:
done